wagtail/contrib/search_promotions/models.py

```diff
@@ -91,26 +143,73 @@ class Meta:
 
 
 class SearchPromotion(models.Model):
+    """
+    Model for search promotion.
+
+    The promotion is for either a page or external url, not both.
+
+    Attributes
+       - query (models.ForeignKey): Matching query object.
+       - page (wagtail.models.Page): Matching page, if any.
+       - external_link_url (models.URLField): Matching external url, if any.
+       - external_link_title (models.CharField) Title text, if external url.
+       - description (models.TextField): Text description of search promotion.
+       - sort_order (int): Ordinal of search promotion.
+    """
+
     query = models.ForeignKey(
         Query, db_index=True, related_name="editors_picks", on_delete=models.CASCADE
     )
     page = models.ForeignKey(
-        "wagtailcore.Page", verbose_name=_("page"), on_delete=models.CASCADE
+        "wagtailcore.Page",
+        verbose_name=_("page"),
+        on_delete=models.CASCADE,
+        null=True,
+        blank=True,
     )
-    sort_order = models.IntegerField(null=True, blank=True, editable=False)
+
+    external_link_url = models.URLField(_("URL"), blank=True)
```

Comment:
```suggestion
    external_link_url = models.URLField(_("external link URL"), blank=True)
```

If we think a descriptive field name is warranted in the code, it’s probably an indication we also should have a descriptive label in the UI?

Reply:
This change has not been applied. I will just suggest we use title case though.

```suggestion
    external_link_url = models.URLField(_("External link URL"), blank=True)
```

Reply:
Sorry but this still has not changed to the full label.